lib/iex/lib/iex/helpers.ex

```diff
@@ -531,6 +534,15 @@ defmodule IEx.Helpers do
     :c.uptime()
   end
 
+  defp print_applications() do
+    Application.started_applications()
```

Comment:
I mean since all the started are loaded, perhaps I could indicate the started ones? Not quite sure if that's useful, but maybe it is. Sometimes teardowns are difficult to debug.